var/spack/repos/builtin/packages/chameleon/package.py

```diff
@@ -11,12 +11,12 @@ class Chameleon(CMakePackage, CudaPackage):
     """Dense Linear Algebra for Scalable Multi-core Architectures and GPGPUs"""
 
     homepage = "https://gitlab.inria.fr/solverstack/chameleon"
-    url = "https://gitlab.inria.fr/solverstack/chameleon/uploads/b299d6037d7636c6be16108c89bc2aab/chameleon-1.1.0.tar.gz"
+    url = "https://gitlab.inria.fr/api/v4/projects/616/packages/generic/source/v1.2.0/chameleon-1.2.0.tar.gz"
     git = "https://gitlab.inria.fr/solverstack/chameleon.git"
     maintainers("fpruvost")
 
     version("master", branch="master", submodules=True)
-    version("1.1.0", sha256="e64d0438dfaf5effb3740e53f3ab017d12744b85a138b2ef702a81df559126df")
+    version("1.2.0", sha256="b8988ecbff19c603ae9f61441653c21bba18d040bee9bb83f7fc9077043e50b4")
```

Comment:
Confirmed the new version sha256.

Normally old versions are deprecated for at least one release (see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions).

Reply:
Ok I keep the old version. I had to ensure the compatibility with starpu (this old version chameleon@1.1.0 is not compatible with starpu@1.4).